src/base/xvega-base.cpp

```diff
@@ -17,23 +17,23 @@ namespace xv
         serialize(j, data.encoding(), "encoding");
 
         int len_selections = data.selections().size();
-        for(int i=0; i<len_selections; i++)
+        /*for(int i=0; i<len_selections; i++)
         {
             xtl::visit([&](auto&& arg){
-                    j["selection"][arg.name().value()]=arg;
+                    j["selection"][arg.name().value()] = arg;
```

Comment:
We should be able to do a normal assignment here

for(int i=0; i<len_selections; i++)
{
    j["selection"][arg.name().value()] = data.selections()[i]
}